### PR TITLE
Add Azure image to alpha/stable channel

### PR DESCRIPTION
**Pull request by kenji-cloudnatix**
Remove the default image hard-coded in populate_instancegroup_spec.go.

### CODE REVIEW

File: channels/alpha [hunk 1]
```diff
@@ -78,6 +78,10 @@ spec:
       providerID: gce
       architectureID: amd64
       kubernetesVersion: ">=1.18.0"
+    - name: "Canonical:UbuntuServer:20.04-LTS:latest"
+      providerID: azure
+      architectureID: amd64
+      kubernetesVersion: ">=1.18.0"
```
Comment (hakman) [inline channels/alpha:84]: Any special reason for 1.18+? I would use the same version as when kOps started supporting Azure.
```suggestion
      kubernetesVersion: ">=1.20.0"
```

Comment (kenji-cloudnatix) [reply channels/alpha:84]: I see. No strong reason. Let me update this and the other channel.

Comment (hakman) [reply channels/alpha]: Great. Thanks. :)

**Review by hakman — APPROVED** [review]
Thanks @kenji-cloudnatix. Should make it easier to get started with Azure.

**Comment by kenji-cloudnatix** [comment]
Great! Thanks for the quick review, @hakman !

**Pull request merged**